docs/concepts/reactiveui/reactive-view-model.md

```diff
@@ -59,3 +59,65 @@ Any change to the view model description property is achieved using the `set` ac
 When _Avalonia UI_ uses the binding to **Update** the view model, the `set` accessor ensures that any parts of the view model that depend on the description property can also react to the change if necessary.
 
 On the next page, you will learn how a reactive command acts as a special case of the view model update.
+
+## Performing Data Validation
```

Comment:
I think that this should have its own page under the ReactiveUI section, especially if this is a standalone example. Can be done in a future PR though.